torch/package/_file_structure_representation.py

```diff
@@ -4,27 +4,47 @@
 from .glob_group import GlobPattern, GlobGroup
 
 
-class Folder:
+class Directory:
+    """A file structure representation. Organized as Directory nodes who have lists of
+    their Directory children."""
+
     def __init__(self, name: str, is_dir: bool):
         self.name = name
         self.is_dir = is_dir
-        self.children: Dict[str, Folder] = {}
+        self.children: Dict[str, Directory] = {}
+
+    def get_dir(self, dirs: List[str]):
+        """Builds path of Directories if not yet built. Return last directory
+        in list.
 
-    def get_folder(self, folders: List[str]):
-        # builds path of folders if not yet built, returns last folder
-        if len(folders) == 0:
+        Args:
+            dirs (List[str]): List of directory names that are treated like a path.
+        """
+        # builds path of directories if not yet built, returns last directory
+        if len(dirs) == 0:
```

Comment:
```suggestion
        if not dirs:
```

Reply:
chiming in randomly, I think `not dirs` is less clear (introduces local ambiguity—are we checking `None`-ness? `False`-ness?)